packages/next/src/export/routes/app-page.ts

```diff
@@ -128,12 +128,23 @@ export async function exportAppPage(
       'utf8'
     )
 
+    const isParallelRoute = /\/@\w+/.test(page)
```

Comment:
Are we prerendering parallel routes, if so why do we need special casing for them at this point? 

Also could we have a test case asserting this behavior possibly?

Reply:
We need the flight data for prefetching or navigation. I’ll check if we have that later

Reply:
add a test case for parallel routes static generation in [7826720](https://github.com/vercel/next.js/pull/63731/commits/782672008745cbba32d1291964d4dff33916b7cd)